pyproject.toml

```diff
@@ -1,8 +1,11 @@
 [build-system]
 # AVOID CHANGING REQUIRES: IT WILL BE UPDATED BY PYSCAFFOLD!
-requires = ["setuptools>=46.1.0,<50", "setuptools_scm[toml]>=5", "wheel"]
+#requires = ["setuptools>=46.1.0,<50", "setuptools_scm[toml]>=5", "wheel"]
+requires = ["setuptools>=46.1.0,<50", "wheel"]
```

Comment:
Let me review whether this change is required.

Reply:
Reverted